README.md

```diff
@@ -10,18 +10,17 @@ being maintained.
 
 The following are basic prerequisites for using Docsy in your site:
 
-- Install a recent release of the Hugo "extended" version (we recommend version
-  0.53 or later). If you install from the [release
```

Comment:
Note that I removed the "(we recommend version  0.53 or later)", leaving at "a recent release" for now.